crates/corro-agent/src/api/peer.rs

```diff
@@ -609,11 +616,13 @@ fn handle_need(
                             last_seq,
                             ts,
                         )? {
+                            // todo: check if we have cleared the row locally
                             sender.blocking_send(SyncMessage::V1(SyncMessageV1::Changeset(
                                 ChangeV1 {
                                     actor_id,
                                     changeset: Changeset::Empty {
                                         versions: empty..=empty,
+                                        ts: Timestamp::from(agent.clock().new_timestamp()),
```

Comment:
this is one of those cases where an empty version is discovered because there are no changes in the crsql_changes. It hasn't cleared it in its bookkeeping table, there's a check for non-null end versions before this section. (we could take the ts in the db anyways even though its not cleared)